services/frontend-v3/src/i18n/en_CA.json

```
@@ -315,6 +315,9 @@
   "profile.visibility.status.public": "Visible to everyone",
   "profile.willing.to.relocate.to": "Willing to relocate to",
   "profile.yes": "Yes",
+  "privacy.modal.header": "Privacy Agreement",
```

To keep consistency with the other titles, Agreement should start with a lower case
```suggestion
  "privacy.modal.header": "Privacy agreement",
```

---

done